### PR TITLE
feat: Add Deploy alias command

### PR DESCRIPTION
**Pull request by kohidave**
Folks might have a hard time finding out that they
can deploy their apps using the CLI. This change
adds an alias to `app deploy` as `deploy` under the
release section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Screen Shot 2020-03-10 at 2 56 09 PM

### CODE REVIEW

File: internal/pkg/cli/app.go [hunk 1]
```diff
@@ -37,3 +37,22 @@ An application represents an Amazon ECS service or task.`,
 	}
 	return cmd
 }
+
+// BuildDeployCmd is the deploy command - which is
+// an alias for app deploy.
+func BuildDeployCmd() *cobra.Command {
```
Comment (efekarakus) [inline internal/pkg/cli/app.go]: nit: should we move this to `deploy.go` and the interfaces in `deploy.go` to `interfaces.go`?

File: internal/pkg/cli/app.go [hunk 2]
```diff
@@ -37,3 +37,22 @@ An application represents an Amazon ECS service or task.`,
...
+	deployCmd := BuildAppDeployCmd()
+	deployCmd.Use = "deploy"
```
Comment (efekarakus) [inline internal/pkg/cli/app.go]: should we add anything to `deployCmd.Aliases`? https://godoc.org/github.com/spf13/cobra#Command

Comment (kohidave) [reply internal/pkg/cli/app.go]: I can't think of any good ones. except maybe `yolo` :P

File: internal/pkg/cli/app.go (hunk 1 above)
Comment (kohidave) [reply internal/pkg/cli/app.go]: yus.

**Pull request merged**